.github/workflows/main.yml

```diff
@@ -20,18 +20,19 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write # Required to obtain the ID token from GitHub Actions
+      contents: read # Required to check out code
```

Comment:
I noticed this one doesn't include the `checks: write # Required for test-reporter` line. Assume that's expected?

Reply:
Yeah, this one doesn't run any unit tests so it doesn't need permissions to report the results back